src/_SignAssembly.targets

```diff
@@ -0,0 +1,32 @@
+<Project>
+  <!--
+      Strong name signing of netstandard assemblies for better cross-runtime compatibility.
+      
+        Ref: https://docs.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/6.0/strong-name-signing-exceptions#recommended-action
+     -->
+
+  <PropertyGroup Condition="'$(SignAssembly)' == 'true'">
+    <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)graphql.snk</AssemblyOriginatorKeyFile>
+    <PublicKeyToken>e6238258560628ee</PublicKeyToken>
+  </PropertyGroup>
+
+
+  <!--
+    Workaround for: https://github.com/Microsoft/msbuild/issues/1490 (lack of support for public-signing satellites)
+    Delay-signs the satellite assemblies when main assembly is public-signed.
+    
+    Ref: https://github.com/dotnet/project-system/blob/main/eng/imports/StrongName.targets
+  -->
+  <Target Name="PrepareToDelaySignSatelliteAssemblies" BeforeTargets="GenerateSatelliteAssemblies" Condition="'$(MSBuildRuntimeType)' != 'Core' and '$(SignAssembly)' == 'true'">
+    <PropertyGroup>
+      <_DelaySignMainAssembly>$(DelaySign)</_DelaySignMainAssembly>
+      <DelaySign Condition="'$(PublicSign)' == 'true'">true</DelaySign>
+    </PropertyGroup>
+  </Target>
+
+  <Target Name="CleanupAfterDelaySigningSatelliteAssemblies" AfterTargets="GenerateSatelliteAssemblies" Condition="'$(MSBuildRuntimeType)' != 'Core' and '$(SignAssembly)' == 'true'">
+    <PropertyGroup>
+      <DelaySign>$(_DelaySignMainAssembly)</DelaySign>
+    </PropertyGroup>
+  </Target>
```

Comment:
@anderslaub Why to bother with this workaround if GraphQL-Parser has no satellite assemblies ?

Reply:
And if so this file can be removed and two lines of config just moved into Directory.Build.props 
```
 <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)graphql.snk</AssemblyOriginatorKeyFile>
 <PublicKeyToken>e6238258560628ee</PublicKeyToken>
```

I'm even not sure if `PublicKeyToken` property is needed since `AssemblyOriginatorKeyFile` contains full public key. How is `PublicKeyToken` property used?

Reply:
I thought I read this was to support `InternalsVisibleTo` but I could be mistaken.

Reply:
#265

Reply:
> @anderslaub Why to bother with this workaround if GraphQL-Parser has no satellite assemblies ?

This is copy-pasted from the referenced link and you're right it is irrelevant to include. We are also not doing PublicSign but real signing.

Reply:
> I'm even not sure if `PublicKeyToken` property is needed since `AssemblyOriginatorKeyFile` contains full public key. How is `PublicKeyToken` property used?

It is not needed for signing the assemblies at all and also not for the friend assembly references (InternalsVisibleTo). 

The property is merely set so it _could_ be used if needed during the build for referencing with or without public key. 

Let me know if I should remove it.

Reply:
Already removed in #365. Let's keep only needed settings in csproj files.